components/prism-elisp.js

```diff
@@ -0,0 +1,192 @@
+// Functions to construct regular expressions
+// simple form
+// e.g. (interactive ... or (interactive)
+function simple_form(name) {
+	return new RegExp("(\\()" + name + "(?=[\\s\\)])")
```

Comment:
Please add semicolons, everywhere needed, in this file. We don't do semicolon-less JS in this repo.